apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/service/NamespaceService.java

```diff
@@ -224,6 +223,7 @@ private NamespaceBO transformNamespace2BO(Env env, NamespaceDTO namespace) {
     //latest Release
     ReleaseDTO latestRelease;
     Map<String, String> releaseItems = new HashMap<>();
+    Map<String, ItemDTO> deletedItemDTOS = new HashMap<>();
```

Comment:
how about deletedItemDTOS => deletedItemDTOs ?

Reply:
Got it.